src/common/constants/constants.js

```diff
@@ -30,27 +35,10 @@ export const INVOICE_STATUSES_OPTIONS = [
 export const PRE_PAY_INVOICE_STATUSES_OPTIONS = INVOICE_STATUSES_OPTIONS
   .filter(option => option.value !== INVOICE_STATUS.paid);
 
-export const PAYMENT_METHOD = {
-  cash: 'Cash',
-  creditCard: 'Credit Card P Card',
-  eft: 'EFT',
-  depositAccount: 'Deposit Account',
-  physicalCheck: 'Physical Check',
-  bankDraft: 'Bank Draft',
-  internalTransfer: 'Internal Transfer',
-  other: 'other',
-};
-
-export const PAYMENT_METHODS_OPTIONS = [
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.cash" />, value: PAYMENT_METHOD.cash },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.creditCard" />, value: PAYMENT_METHOD.creditCard },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.eft" />, value: PAYMENT_METHOD.eft },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.depositAccount" />, value: PAYMENT_METHOD.depositAccount },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.physicalCheck" />, value: PAYMENT_METHOD.physicalCheck },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.bankDraft" />, value: PAYMENT_METHOD.bankDraft },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.internalTransfer" />, value: PAYMENT_METHOD.internalTransfer },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.other" />, value: PAYMENT_METHOD.other },
-];
+export const PAYMENT_METHOD_OPTIONS = Object.values(PAYMENT_METHOD).map(method => ({
```

Comment:
isn't it the same in different modules? I thought you've added this to acq-components also

Reply:
I added options for `<FieldSelect>` where we use `labelId` instead `label`

Reply:
But you have it in stripes-acq-components, no? https://github.com/folio-org/stripes-acq-components/blob/1ffbf932fce0ed41e28bf1486953e1946e9c449a/lib/constants/paymentMethod.js#L37

Reply:
[acqCheckBoxFilter](https://github.com/folio-org/stripes-acq-components/pull/71) was extended to use these options